packages/plugin/src/generators/app/templates/src/widgets/SampleWidget/WidgetProperties.json

```diff
@@ -6,10 +6,10 @@
     "src": "src/widgets/SampleWidget",
     "properties": [
         {
-            "id": "message",
-            "name": "Message",
+            "name": "message",
             "type": "text",
-            "optional": true
+            "isOptional": true,
+            "inputLabel": "Message"
```

Comment:
What is this used for, and how is it different from `displayName`?

Reply:
You're right. Should be displayName. I'll change it.